packages/sedlex/sedlex.3.1/opam

```diff
@@ -0,0 +1,49 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "An OCaml lexer generator for Unicode"
+description: """
+sedlex is a lexer generator for OCaml. It is similar to ocamllex, but supports
+Unicode. Unlike ocamllex, sedlex allows lexer specifications within regular
+OCaml source files. Lexing specific constructs are provided via a ppx syntax
+extension."""
+maintainer: ["Alain Frisch <alain.frisch@lexifi.com>"]
+authors: [
+  "Alain Frisch <alain.frisch@lexifi.com>"
+  "https://github.com/ocaml-community/sedlex/graphs/contributors"
+]
+license: "MIT"
+homepage: "https://github.com/ocaml-community/sedlex"
+bug-reports: "https://github.com/ocaml-community/sedlex/issues"
+depends: [
+  "ocaml" {>= "4.14"}
+  "dune" {>= "2.8"}
+  "ppxlib" {>= "0.26.0"}
+  "gen"
+  "uchar"
```

Comment:
Is there a reason why `uchar` is still required?

Reply:
None.. Removed, thanks!